gocumber_test.go

```diff
@@ -140,11 +140,60 @@ func TestRun_SuccessWithOutlineSteps(t *testing.T) {
 	tt := new(testing.T)
 
 	steps.Given("I have no users", func([]string, StepNode) {})
-	steps.When("I create a new user with the following data:", func([]string, StepNode) {})
 	steps.Then("no users should be created", func([]string, StepNode) {})
 
+	var expected_first_example = map[string]string{
+		"key":      "value",
+		"username": "",
+		"first":    "",
+		"last":     "",
+	}
+
+	var expected_second_example = map[string]string{
+		"key":      "value",
+		"username": "",
+		"first":    "fname",
+		"last":     "lname",
+	}
+
+	var expected_third_example = map[string]string{
+		"key":      "value",
+		"username": "newuser",
+		"first":    "",
+		"last":     "lname",
+	}
+
+	var expected_fourth_example = map[string]string{
+		"key":      "value",
+		"username": "newuser",
+		"first":    "fname",
+		"last":     "",
+	}
+
+	var expected_fifth_example = map[string]string{
+		"key":      "value",
+		"username": "newuser",
+		"first":    "fname",
+		"last":     "lname",
+	}
+
+	var expected_replaced_table_data = []map[string]string{
+		expected_first_example,
+		expected_second_example,
+		expected_third_example,
+		expected_fourth_example,
+		expected_fifth_example,
+	}
+
+	var called int
+	steps.When("I create a new user with the following data:", func(_ []string, step StepNode) {
+		assert.Equal(t, expected_replaced_table_data[called], ColumnMap(step.Table()))
```

Comment:
Sure, no worries. Addressed in https://github.com/sittercity/gocumber/commit/b2e3703e941199b22a3afdaf490724191ba60090